### ISSUE TITLE
add alex skin format. We only have steve

### ISSUE THREAD

**Issue by aydanmassa** [LAX1DUDE/eaglercraft#96]

**Comment by LAX1DUDE** [comment]
yeah, that wouldn't be too bad to implement, the one problem is redesigning the profile editor screen to include a checkbox when you add a skin to set it to add the skin in the Alex format. It would be hard to squeeze that into the current profile screen in a visually appealing format, but it probably wouldn't be too hard to just provide an initial popup including the option to use the Alex model when you click the Add Skin button before it displays the actual file chooser.

**Comment by aydanmassa** [comment]
> yeah, that wouldn't be too bad to implement, the one problem is redesigning the profile editor screen to include a checkbox when you add a skin to set it to add the skin in the Alex format. It would be hard to squeeze that into the current profile screen in a visually appealing format, but it probably wouldn't be too hard to just provide an initial popup including the option to use the Alex model when you click the Add Skin button before it displays the actual file chooser.

awesome
!
